src/main/java/se/knowit/bookitregistration/service/exception/ConflictingEntityException.java

```diff
@@ -0,0 +1,21 @@
+package se.knowit.bookitregistration.service.exception;
+
+public class ConflictingEntityException extends Exception {
+
```

Comment:
A serialVersionUID should be added for exceptions see https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it